src/components/LocationAutocomplete/LocationAutocomplete.tsx

```diff
@@ -11,12 +10,47 @@ import POWERED_BY_GOOGLE_ON_WHITE_2X from './images/powered_by_google_on_white@2
 import POWERED_BY_GOOGLE_ON_WHITE_3X from './images/powered_by_google_on_white@3x.png';
 import styles from './LocationAutocomplete.scss';
 
+interface Props {
+    /** input field ref */
+    inputRef?: React.RefObject<HTMLInputElement>;
+    /** to be shown in the input field when no value is typed */
+    inputPlaceholder: string;
+    /** default input value */
+    defaultInputValue?: string;
+    /** to be shown when no suggestions are available */
+    noSuggestionsPlaceholder: string;
+    /** trigger of the initial focus of the input field */
+    isFocused?: boolean;
+    /** label for the Clear button */
+    clearLabel?: string;
+    /** defines if there's a single location to select in component */
+    singleLocation?: boolean;
+    /** callback to be called with selected value. */
+    onSelectionChange: (value: google.maps.places.AutocompletePrediction) => void;
```

Comment:
It would be nice to have a rule of thumb to expose Types of arguments of the callbacks. For instance, if I want to define `onSelectionChange` handler outside of the render method:
```
const handlerSelectionChange = (args: onSelectionChangeLocationAutocomplete) => {
....
}
render (
  <LocationAutocomplete onSelectionChange={handlerSelectionChange} ...>
);
```
What do you think?

Reply:
yes, I agree! Only, I'm not sure we ever set it up properly (exposing TS types - https://jira.textkernel.nl/browse/ONEUI-213). If there is a pattern I should apply to make types exposed, that is already working, please help me out with it.

Reply:
For `LocationSelector` we already export component Props Types and helper utils Types, but probably it isn't a good example. We can wait for ONEUI-213 and then discuss it in detail about expose Types rules.